cds/modules/announcements/alembic/36f19890d4fd_create_announcements_branch.py

```diff
@@ -0,0 +1,38 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2018 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Creation of announcements table"""
+
+# revision identifiers, used by Alembic.
+revision = '36f19890d4fd'
+down_revision = None
+branch_labels = (u'announcements',)
```

Comment:
and here the branch label `cds_videos`. The point is to have one branch for cds-videos and then after all the recipes for videos.

Reply:
If we would have a CDS specific DB model I would do it like this, but since this can potentially be extracted to its own module ... I don't know it seems to be more natural to have a branch for the announcements, lets see what the others think.

Reply:
if it may be a different module I agree to leave it like this.